src/components/LeafletMap.js

```diff
@@ -55,6 +55,20 @@ const activeInnerMarkerHtmlStyles = `
   border-top: 15px solid ${defaultMarkerColor};
 `;
 
+const clusterIconStyles = `
+  width: 24px;
+  height: 24px;
```

Comment:
Making a tagged litteral higher-order function might be the way to go here, if you want to make the size and shape (or even the colour) of the cluster variable in itself. I'm going to try to set up an example.

Reply:
I'm going to talk about this in #javascript, because this is a tricky topic that involves higher order functions.